include/xsimd/config/xsimd_arch.hpp

```diff
@@ -184,7 +184,8 @@ namespace xsimd
     {
     };
     using all_x86_architectures = arch_list<avx512bw, avx512dq, avx512cd, avx512f, fma3<avx2>, avx2, fma3<avx>, avx, fma4, fma3<sse4_2>, sse4_2, sse4_1, /*sse4a,*/ ssse3, sse3, sse2>;
-    using all_arm_architectures = arch_list<detail::sve, neon64, neon>;
+    using all_sve_architectures = arch_list<detail::sve<512>, detail::sve<256>, detail::sve<128>>;
+    using all_arm_architectures = typename detail::join<arch_list<neon64, neon>, all_sve_architectures>::type;
```

Comment:
Will it cause default arch to pick NEON, even if SVE related compiler options are enabled? If compiler supports SVE, it must also support NEON.

Reply:
Correct!